### ISSUE TITLE
Insufficient color contrast in footer

### ISSUE THREAD

**Issue by alexdryden** [UIUCLibrary/arclight#20]
See: https://github.com/UIUCLibrary/arcuit/pull/7 for context

Footer “Give” button – The color contrast ratio doesn’t meet accessibility requirements. One option can be to replace the Illini Orange with Altgeld Orange in the footer component:
<style> ilw-footer{ --il-orange: #C84113; } </style>
